Missing includes

https://github.com/HeavyIonAnalysis/QnAnalysis/issues/16

> Hi,
>My installation throws errors during the compilation:

    QnAnalysis/build/_deps/qntools-src/include/base/QnTools/DataContainer.hpp:407:12: error: no member named 'set' in namespace 'std'

> Proposed solution:
> add #include <set> in DataContainer.hpp and ~~#include <TObjString.h> in CorrelationTaskRunner.cpp~~

--- a/include/base/QnTools/DataContainer.hpp
+++ b/include/base/QnTools/DataContainer.hpp
@@ -22,6 +22,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <set>
 
 #include "Axis.hpp"
 #include "DataContainerHelper.hpp"

